Source/DataSources/Entity.js

```diff
@@ -362,6 +366,12 @@ import WallGraphics from './WallGraphics.js';
          * @type {ModelGraphics}
          */
         model : createPropertyTypeDescriptor('model', ModelGraphics),
+        /**
+         * Gets or sets the tileset.
+         * @memberof Entity.prototype
+         * @type {TilesetGraphics}
```

Comment:
There's no such type as `TilesetGraphics`. This should be `Cesium3DTilesetGraphics`. This produces an unclickable type in the doc here: http://localhost:8080/Build/Documentation/Entity.html#tileset

Reply:
Yes, though party of me thought about removing the Cesium3D prefix from everywhere, but I guess keeping the two consistent makes more sense.